src/discovery/azure/Akka.Discovery.Azure/ClusterMemberTableClient.cs

```diff
@@ -0,0 +1,204 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="ClusterMemberTableClient.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Akka.Annotations;
+using Akka.Discovery.Azure.Model;
+using Akka.Event;
+using Azure;
+using Azure.Data.Tables;
+
+namespace Akka.Discovery.Azure
+{
+    [InternalApi]
+    public class ClusterMemberTableClient
```

Comment:
Does this need to be `public` or can we make it `internal`?

Reply:
Don't they all get started internally by the `Akka.Discovery` plugin though? i.e. the user doesn't have to launch any of those components.

Reply:
Right, I'll change it.

Reply:
Done